File: 944. Delete Columns to Make Sorted/main.ts
class Solution {
  minDeletionSize(strs: string[]): number {
    let countCol = 0;
    for (let i = 0; i < strs[0].length; i++) {
      let prev = strs[0][i];
      let curr = "";
      for (let j = 1; j < strs.length; j++) {
        curr = strs[j][i];
        if (prev > curr) {
          countCol++;
          break;
        } else {
          prev = curr;
        }
      }
    }
    return countCol;
    }
}

File: 944. Delete Columns to Make Sorted/main.py
class Solution(object):
    def minDeletionSize(self, strs):
        """
        :type strs: List[str]
        :rtype: int
        """
        count_col = 0
        for i in range(len(strs[0])):
            prev = strs[0][i]
            curr = ''
            for j in range(1,len(strs)):
                curr = strs[j][i]
                if ord(prev) > ord(curr):
                    count_col+=1
                    break
                else:
                    prev = curr
        return count_col

            
inst_tst = Solution()


print(inst_tst.minDeletionSize(["rrjk","furt","guzm"]))